### PR TITLE
fix: applying special character exceptions for workspace names

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image
image

### CODE REVIEW

File: apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue [hunk 1]
```diff
@@ -16,16 +16,38 @@ const props = withDefaults(defineProps<Props>(), {
     size: 'sm',
 });
 
+const ENGLISH_REGEX = /^[A-Za-z]/;
+const CORPORATE_REGEX = /\(([^)]+)\)/;
```
Comment (yuda110) [inline apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue:20]: praise: cool!!!

File: apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue [hunk 2]
```diff
@@ -16,16 +16,38 @@ const props = withDefaults(defineProps<Props>(), {
...
+
 const state = reactive({
     theme: computed(() => props.theme),
-    logoText: computed(() => props.text.slice(0, 1).toUpperCase()),
-    isLogoStartsWithEnglish: computed(() => /^[A-Za-z]/.test(state.logoText)),
+    temp: computed(() => {
+        const regex = /\(([^)]+)\)/;
+        return regex.exec(props.text);
+    }),
```
Comment (yuda110) [inline apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue:27]: quibble: temp!?🧐

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

**Review by piggggggggy — APPROVED** [review]
LGTM~

File: apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue (hunk 2 above)
Comment (skdud4659) [reply apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue:27]: ㅇㅅㅇ

**Pull request merged**